snforge_std/src/_cheatcode.cairo

```diff
@@ -18,7 +18,8 @@ pub fn handle_cheatcode(input: Span<felt252>) -> Span<felt252> {
     }
 }
 
-fn _is_config_run() -> bool {
+// Do not use this function directly.
```

Comment:
Please describe that it is an internal part of the snforge architecture used by macros. `Do not use this function directly.` raises a question: "Why?".